gcal_sync/model.py

```diff
@@ -343,6 +343,9 @@ class Attendee(CalendarBaseModel):
     optional: bool = False
     """Whether this is an optional attendee."""
 
+    isSelf: bool = False
```

Comment:
Please change to is_self to match the style of other fields in this file. Thank you!